src/Command/InfectionCommand.php

```diff
@@ -253,6 +260,38 @@ private function getOutputFormatter(): OutputFormatter
         throw new \InvalidArgumentException('Incorrect formatter. Possible values: "dot", "progress"');
     }
 
+    /**
+     * By default we let a mutant process use as much more memory as an initial test suite consumed.
+     *
+     * @var int
+     */
+    const DEFAULT_MEMORY_LIMIT_MULTIPLIER = 2;
```

Comment:
imo all constants should be placed at the top of the class

Reply:
1. This isn't going to make the code more readable. 
2. PHP CS Fixer normally takes care of this if there's such policy, and it didn't.

When there's no policy, I side with *Code Complete* on minimizing live time for programming constructs.

Reply:
if you find any place in the infection code where we did it in the same way - feel free to leave it as is. if no - please move it to the top of the class. thank you

Reply:
However you wish. Done.